include/sys/asm_linkage.h

```diff
@@ -28,13 +28,8 @@
 #define	_SYS_ASM_LINKAGE_H
 
 #if defined(__i386) || defined(__amd64)
-
-#ifdef _WIN32
-#include <win/asm_linkage.h>
-#else
 #include <sys/ia32/asm_linkage.h>	/* XX64	x86/sys/asm_linkage.h */
 #endif
-#endif
```

Comment:
Just undoing some previous work, we had win/asm_linkage.h to be different to linux